cajita/unit_test/tstLocalMesh.hpp

```diff
@@ -613,18 +611,24 @@ void irregularTest( const std::array<int, 3> &ranks_per_dim )
                      k_func( own_cell_global_space.max( Dim::K ) ) );
 
     EXPECT_FLOAT_EQ( ghost_lc_m( Dim::I ),
-                     i_func( ghost_cell_global_space.min( Dim::I ) ) );
+                     i_func( own_cell_global_space.min( Dim::I ) -
+                             own_cell_local_space.min( Dim::I ) ) );
```

Comment:
Does it make sense to add a function to return ghost low and high corners for convenience?

Reply:
The new local-to-global indexer in #251 should do this.

Reply:
Yep, just realized that when I went to the next one